intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md

```diff
@@ -0,0 +1,46 @@
+# icnet-camvid-fp32-0001
```

Comment:
Add `Legal information` part (see other model's description for example)

Reply:
Done